corehq/util/htmx_action.py

```diff
@@ -117,8 +123,7 @@ def dispatch(self, request, *args, **kwargs):
         try:
             response = handler(request, *args, **kwargs)
         except HtmxResponseException as err:
-            context = self.get_htmx_error_context(**kwargs)
-            context["htmx_error"] = err
+            context = self.get_htmx_error_context(action, err, **kwargs)
```

Comment:
This is only for deliberately raised exceptions, right?  IIRC, an uncaught exception will return a generic 500 error modal, does that behavior remain?

Reply:
yes, this is only for deliberately raised exceptions